docs/semgrep-sc/sc-getting-started.md

```diff
@@ -0,0 +1,12 @@
+---
+slug: scanning-open-source-dependencies
+append_help_link: true
+description: "Scan your codebase's open-source dependencies with Semgrep Supply Chain's high-signal rules that determine a vulnerability's reachability."
+tags:
+    - Semgrep Supply Chain
+    - Team & Enterprise Tier
+title: Scanning open-source dependencies
```

Comment:
```suggestion
title: Scanning open source dependencies
```

I am not sure if we formalized it but I think we have been standardizing on “open source” instead of “open-source.”